### PR TITLE
WIP: Overhaul EditEvent Userinterface

### PR DESCRIPTION
**Pull request by newhinton**

### CODE REVIEW

File: res/layout/edit_event.xml [hunk 1]
```diff
@@ -70,19 +78,33 @@
...
                 android:layout_marginStart="0dip"
                 android:layout_marginTop="0dip"
-                android:layout_gravity="start|top"/>
+                android:visibility="gone" />
         </FrameLayout>
     </ScrollView>
-</LinearLayout>
+
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```
Comment (jspricke) [inline res/layout/edit_event.xml:99]: why this?

File: res/menu/event_info_title_bar.xml [hunk 2]
```diff
@@ -20,14 +20,14 @@
         android:alphabeticShortcut="s"
         android:title="@string/share_label"
         android:icon="@drawable/ic_share_event"
-        app:showAsAction="always" />
+        app:showAsAction="ifRoom" />
```
Comment (jspricke) [inline res/menu/event_info_title_bar.xml:23]: why this? Dito the two below.

**Comment by newhinton** [comment]
@Gitsaibot could you provide a screenshot how google does it?

I think in the current form, the save&cancel actions take way to much space for it's function

File: res/menu/event_info_title_bar.xml (hunk 2 above)
Comment (newhinton) [reply res/menu/event_info_title_bar.xml:23]: They are not often used, and create a cluttered ui. We should show only often used and important actions, and hide the rest. Showing 4 or more icons in the topbar is to much and unnessesssary.

File: res/layout/edit_event.xml (hunk 1 above)
Comment (newhinton) [reply res/layout/edit_event.xml:99]: See here:
https://github.com/Etar-Group/Etar-Calendar/pull/736#issuecomment-659226086

**Comment by newhinton** [comment]
I added the X button (it may be too big) and coloured the fab aswell

**Comment by newhinton** [comment]
@Gitsaibot why? it hasnt been merged, and i still think that etar could use an overhaul

**Comment by newhinton** [comment]
@Gitsaibot No i did not see your changes.

 I just checked them out, and i have to say, your spacer are waaay better than mine. I also like the icons.
I still think the topbar could be switched to the one i am using, which frees up a bit more space. If you want me to, i can make a pr for it, but this one can be closed i guess

**Pull request closed without merging**